chcemvediet/apps/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 
 class ChcemvedietConfig(AppConfig):
-    name = u'chcemvediet.apps'
```

Comment:
@viliambalaz Si si to lokalne u seba testoval, ci to funguje?

Reply:
Testoval som to priamo cez admin rozhranie, neuvedomil som si ze som spravil takyto zasah.

Reply:
@viliambalaz Aha, admin mozno siel. Ale klasicky web nesiel, lebo nenaslo templaty pre web. Kedze ich hladalo pod inou app.